src/planwise/client/components/nav.cljs

```diff
@@ -3,12 +3,17 @@
 
 ;; Navigation components
 
-(defn- li-menu-item [{:keys [selected item href title icon]}]
+(defn- li-menu-item [{:keys [selected item href title icon tab-number wizard-state]}]
   (let [is-selected? (or (= selected item)
                          (item selected))]
-    [:li {:class (when is-selected? "active")}
+    [:li {:class (str
+                  (when is-selected? "active ")
+                  (when wizard-state (str (name wizard-state) " wizard")))}
      [:a {:href href}
       (some-> icon (components/icon "icon-small"))
+      (when (and wizard-state (> tab-number 1)) [:div.separator])
```

Comment:
Are you referring to the div or the class? What do you mean by redundant? It's part of the design.